zephyr/lib/regions_mm.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Jakub Dabek <jakub.dabek@intel.com>
+ */
+
+#include <sof/lib/regions_mm.h>
+
+struct virtual_memory_heap
+	vm_heaps[CONFIG_MP_MAX_NUM_CPUS + VIRTUAL_REGION_COUNT] = { {0} };
```

Comment:
this initialisation is also redundant - global and static data is always initialised to 0 by default